### PR TITLE
Clean Back-office templates, part 2 - configure

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | See below.
| Type?             | improvement
| Category?         | BO
| BC breaks?        | This PR should be considered as a BC because of lot of HTML structure changes and they might be used by a module or anything else, but it's acceptable. The optimizations are only within the files itself, so it does not matter if anyone uses the old version in any form in their modules.
| Deprecations?     | no
| How to test?      | Go through Shop parameters and Advanced parameters sections in BO and see if nothing is broken by this PR. There should be no visible change.
| Possible impacts? | No bad impacts. :-)

### Description
- I went through all templates and removed 99% of unneeded boostrap code like `<div class="row"><div class="col">XXX</div></div>`. It will reduce render depth and complexity by several layers - easier to navigate, debug CSS, maybe faster to render on slower PCs.
- I fixed all places where somebody used single quotations or no quotations at all.
- I fixed all places where somebody combined row and column in same element, this is not allowed.
- I fixed all places where somebody used syntax line col-sm-12 col-md-12 or col-md-12 col-lg-6. The first one is not needed.
- Small fixes here and there - removed some container classes, fixed module position page responsivity a bit, fixed virtual product block responsivity etc...
It should produce no BC breaks, unless some 3rd party module was targetting some classes they shouldn't have OR they targeted it by child or parent.

### Templates concerned
Configure/AdvancedParameters
Configure/AdvancedParameters/Backup
Configure/AdvancedParameters/Email
Configure/AdvancedParameters/Employee
Configure/AdvancedParameters/FeatureFlag
Configure/AdvancedParameters/ImportDataConfiguration
Configure/AdvancedParameters/ImportPage
Configure/AdvancedParameters/LogsPage
Configure/AdvancedParameters/Profiles
Configure/AdvancedParameters/RequestSql
Configure/AdvancedParameters/Webservice
Configure/ShopParameters
Configure/ShopParameters/Contact/Contacts
Configure/ShopParameters/OrderPreferences
Configure/ShopParameters/OrderReturnStates
Configure/ShopParameters/OrderStates
Configure/ShopParameters/TrafficSeo/Meta
Configure/ShopParameters/TrafficSeo/SearchEngines

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26061)

### CODE REVIEW

**Review by NeOMakinG — DISMISSED** [review]
QA will cry 😄 

@Julievrz same, no wording changes

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig:30]: Useless line here it seems

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig:116]: Same here

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/domain_name_management.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/domain_name_management.html.twig:37]: Extra line here

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/domain_name_management.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/domain_name_management.html.twig:43]: And here

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/shop_urls_configuration.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/shop_urls_configuration.html.twig:40]: Extra line

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/shop_urls_configuration.html.twig
Comment (jolelievre) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/shop_urls_configuration.html.twig:46]: Extra line

**Review by jolelievre — DISMISSED** [review]
Thank you @Hlavtox 
Just a few comments about some extra lines that could be cleaned, but it's not blocking Feel free to decide if it's worth cleaning

**Comment by hibatallahAouadni** [comment]
> @Julievrz same, no wording changes

Ping @NeOMakinG should we remove the label **`Waiting for wording`**?

**Comment by NeOMakinG** [comment]
> > @Julievrz same, no wording changes
> 
> Ping @NeOMakinG should we remove the label **`Waiting for wording`**?

Yep as @Hlavtox didn't change any wording

**Comment by khouloudbelguith** [comment]
Hi @Hlavtox,

Can you please check the Conflicting files?

Thanks!

**Comment by Hlavtox** [comment]
@khouloudbelguith Rebased, QA please? :-)

**Comment by khouloudbelguith** [comment]
Hi @Hlavtox,

This PR should be reviewed by the Developers.

>Sorry, I updated my comment => should be reviewed first

Thanks!

**Comment by Hlavtox** [comment]
@khouloudbelguith It was 8 lines of code and I did it manually by comparing the files one by one...

**Review by NeOMakinG — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
> @khouloudbelguith It was 8 lines of code and I did it manually by comparing the files one by one...

This is the rule :p

**Review by Progi1984 — DISMISSED** [review]

**Comment by hibatallahAouadni** [comment]
@jolelievre we need your approval to change the status to **Waiting for QA** :sweat_smile:

**Review by jolelievre — DISMISSED** [review]
Thanks @Hlavtox

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

Thanks for your huge PR :clap: 
LGTM, except when I launch the automated test on this Maintenance page, check the screen record:

https://www.awesomescreenshot.com/video/5868563?key=e048ffee5953900ef2be184bc927bdf3

When you try to disable the shop while the debug mode enabled, the FO crashed!

image

image

But when you disable the debug mode, you can see the FO Maintenance mode well displayed but the logo is not displayed!
If you check the src of logo, you're gonna see: `src(unknown)`
It will be fixed if we change this line: https://github.com/PrestaShop/PrestaShop/blob/8d63d3a3323d18c7535c8b0f9d4455958cfbe07a/themes/classic/templates/errors/maintenance.tpl#L34
By: `src={$shop.logo.src}`
Well, IMHO I think this should be fixed on `1.7.8.x` and `develop`, it seems to me this bug is not related to your PR.
But, I don't know why without your PR, it works :sweat_smile: 

PS: @PrestaShop/prestashop-maintainers wdyt?
Please check and feedback.

Thanks!

**Comment by jolelievre** [comment]
Thanks @hibatallahAouadni 

Indeed I don't think the bug is related with this PR, did you try on a fresh `1.7.8.x` or `develop` branch to make sure of it? If the bug existed previously then it's not related to this PR and we need to open an issue (the severity and version target will be decided by the product team I guess).

When you mention the `automated test` that you launched for the maintenance page do you mean the ones from the nightly test suite? Or the ones from the CI suite? (sanity tests)

If it's the ones from nightly then they should have already failed no? 🤔

**Comment by Hlavtox** [comment]
@hibatallahAouadni @jolelievre The code is still there on develop branch, not an issue with this PR ;-)

**Comment by hibatallahAouadni** [comment]
Hello @jolelievre 

> Indeed I don't think the bug is related with this PR, did you try on a fresh 1.7.8.x or develop branch to make sure of it?

Yes, I did and failed to reproduce the issue with PS178x and develop :disappointed: But @khouloudbelguith reproduced it **once** randomly and after clearing the cache and reinstalling the store, the issue has been faded. 
In my case, even when I clear the cache multiple times and redo the `composer install`, `make install` and reinstall the shop, the issue is always there.

> When you mention the automated test that you launched for the maintenance page do you mean the ones from the nightly test suite? Or the ones from the CI suite? (sanity tests).

Functional test https://github.com/PrestaShop/PrestaShop/tree/develop/tests/UI/campaigns/functional/BO (13_shopParameters & 14_advancedParameters).

Thanks!

**Comment by hibatallahAouadni** [comment]
@Hlavtox could you please rebase your PR :pray:

**Comment by Hlavtox** [comment]
@hibatallahAouadni rebased

**Review by jolelievre — APPROVED** [review]

**Comment by hibatallahAouadni** [comment]
Hello @Hlavtox 

Thanks for your patience :pray: 
LGTM, QA :heavy_check_mark: 

PS: the only issue I encountred while auto tests is #26463 (and it's has been fixed on develop :tada: )

Thanks!

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Thanks @Hlavtox & @hibatallahAouadni

**Pull request merged**
